frontend/README.md

```diff
@@ -1,32 +1,33 @@
+<p align="center">
+    <a href="https://airy.co/">
+    <img width="350" src="./assets/airy_primary_rgb.svg" alt="Airy Logo" />
+    </a>
+</p>
+
+# [Airy](https://airy.co/)- A Conversational Data Platform to drive sales, improve marketing and provide better service
+
+
 # Frontend
 
-The `frontend` top-level folder of the Airy Core Platform
-[monorepo](https://en.wikipedia.org/wiki/Monorepo) contains the UI related code
-of the platform.
+The Frontend section of the Airy core platform contains the user interaction(UI) related code.
 
-Here is a quick introduction to the frontend projects:
+They are grouped into four frontend sub-projects:
 
-- `components`
+- `Components` [![npm version](https://badge.fury.io/js/name-suggestion-index.svg)](https://www.npmjs.com/package/@airyhq/components)
 
-  Here you can find the [Airy
-  Components Library](https://www.npmjs.com/package/@airyhq/components library. We are open-sourcing
-  (it's a work in progress) the critical components necessary to build a
-  messaging application. We use this library to power our commercial offering.
+  Here you can find the Airy Component Library. We are open-sourcing the critical components necessary to build a messaging application (it's a work in progress). We use this library to power our commercial offering.
 
-- `showcase`
+- `Showcase`
 
   This directory contains a small React application that showcases the
-  components of the [Airy Components
-  Library](https://www.npmjs.com/package/@airyhq/components), we host the `main`
-  branch version at [https://components.airy.co](https://components.airy.co).
+  components of the [Airy Components Library](https://www.npmjs.com/package/@airyhq/components), we host the `main`
+  branch version [here](https://components.airy.co).
```

Comment:
Since we're touching this.... this is actually out of date as the components and the show case library are not there anymore so we should just remove all together.